src/rgw/rgw_pubsub_push.cc

```diff
@@ -314,6 +320,7 @@ RGWPubSubEndpoint::Ptr RGWPubSubEndpoint::create(const std::string& endpoint,
       throw configuration_error("unknown amqp version " + version);
       return nullptr;
     }
+#endif
```

Comment:
probably better to put the amqps case into the ifdef

Reply:
done.